10309.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright 2018 Hua Xiang IOT. All rights reserved.
+
+-->
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>DeviceDataContainer</Name>

```

Comment:
The name is quite generic for a relatively specific object

Reply:
We wanted to keep some information about the device, so I blurred the name.We're sorry about that.We will uphold an open and cooperative attitude to further improve the object.
Therefore, we change the name of the object to ShareParkingLock.
The lock is used to control each parking space and upload the lock information via wireless network.